### ISSUE TITLE
Bug teleporters

### ISSUE THREAD

**Issue by brenoepics** [brenoepics/morningstar#87]
In GitLab by @KittMustang on May 16, 2019, 21:13

## Build Information
Emulator Version: Arcturus RC-2

## Expected Behavior
In this video I put two teleporters in my room, one of which is on the door. The problem is that I enter the room and the character teleports to the other tile. This should NOT happen. Sometimes the teleporter (if placed between two rooms) goes in a continuous loop between teleporter number 1 and teleportation number 2.
Simply put: on Habbo the teleporter works even if placed in the door on the room.
## Current Behavior
Teleportation is also activated in the room door.

## Steps to Reproduce
Follow these two videos
- 1: https://www.youtube.com/watch?v=MpNc0uNIIeA
- 2: https://www.youtube.com/watch?v=D30yy6Pht4U (other teleportation bug (it happens more rarely than the previous bug).
